src/common/tab/tabView.component.tsx

```diff
@@ -7,64 +7,93 @@ import { useSearchParams } from 'react-router-dom';
 import { a11yProps, StyledTab } from '../../common/tab/tab.utils';
 import TabPanel from '../../common/tab/tabPanel.component';
 
+type AdditionalTabValues = 'Gallery' | 'Attachments';
+
 export interface TabViewProps<T> {
```

Comment:
```suggestion
export interface TabViewProps<T extends string> {
```
should mean that you can remove all of the `as unknown as string` castings. Similar change could be made to `a11yProps`